content/participants/ttjc@swarajpure/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Swaraj Rajpure
+type: "participant"
+name: "Swaraj Rajpure"
+portfolio: ""
+github-link: "https://github.com/tanaypratap"
+linkedin-link: "https://github.com/swarajpure"
+looking-for: "internship"
+description: "learning the"
```

Comment:
Hey! @swarajpure I think you left this incomplete.

Reply:
Hello. Should I create a new PR fixing this?